src/Data/Nat/Properties.agda

```diff
@@ -1229,6 +1247,14 @@ m⊔n≤m+n m n with ⊔-sel m n
 ... | inj₁ m⊓n≡m rewrite m⊓n≡m = m≥o
 ... | inj₂ m⊓n≡n rewrite m⊓n≡n = n≥o
 
+⊓-least : ∀ {m n o} → m ≤ o ⊎ n ≤ o → m ⊓ n ≤ o
+⊓-least {zero}  {n}     (inj₁ m≤o)       = z≤n
+⊓-least {suc m} {zero}  (inj₁ m≤o)       = z≤n
+⊓-least {suc m} {suc n} (inj₁ (s≤s m≤o)) = s≤s (⊓-least (inj₁ m≤o))
+⊓-least {zero}  {n}     (inj₂ n≤o)       = z≤n
+⊓-least {suc m} {zero}  (inj₂ n≤o)       = n≤o
+⊓-least {suc m} {suc n} (inj₂ (s≤s n≤o)) = s≤s (⊓-least (inj₂ n≤o))
+
```

Comment:
Same comment about `Selective` simplifying the proof.

Reply:
Again we've already proved the two parts.